src/general/g.cloud/cloud_unpack.py

```diff
@@ -44,8 +44,8 @@
 logging.debug("Unpacking in <%s/%s/%s>" % (dbase, location, mapset))
 f.write("Unpacking in <%s/%s/%s>" % (dbase, location, mapset))
 
-# read for grass70 executable the path to gisbase
-for line in open(which.which("grass70")).readlines():
+# read for grass executable the path to gisbase
+for line in open(which.which("grass")).readlines():
     if line.startswith('    gisbase = "'):
         gisbase = line.split("=")[-1].split('"')[1]
```

Comment:
This might be outdated as we nowadays have:

`grass --config path --tmp-location epsg:4326 --exec g.gisenv get=GISBASE`

Maybe to be addressed in a separate PR.

Reply:
Ok, we should take note of it for another PR

Reply:
For now a new issue: #668

Reply:
> I couldn't edit the PR, nor a fork of it, so I followed the flow of the github web UI, and you have a PR on your branch for g.cloud

Thanks, merged into this PR.